admin/test/test_acceptance.py

```diff
@@ -205,3 +208,27 @@ def test_non_json_message(self):
             )],
             self._convert(NON_JSON_JOURNAL_EXPORT),
         )
+
+
+class CommonOptionsTests(TestCase):
+    """
+    Tests for ``CommonOptions``.
+    """
+    def test_invalid_version(self):
+        """
+        An ``UsageError`` is thrown if an invalid flocker version is passed
+        to the ``CommonOptions``.
+        """
+        arg_options = (
+            "--distribution ubuntu-14.04 --provider AWS "
+            "--flocker-version invalid-version"
+        )
+        common_options = CommonOptions(self.mktemp())
+        exception = self.assertRaises(
+            UsageError,
+            common_options.parseOptions, arg_options.split()
+        )
+
+        self.assertIn(
+            "Flocker version", exception.args[0]
```

Comment:
:question: This doesn't seem specific enough. Imagine if command line errors always printed the ``--help`` and if the ``--help`` included the text ``Flocker Version``.